sdks/python/scripts/run_integration_test.sh

```diff
@@ -281,23 +280,12 @@ fi
 # Run tests and validate that jobs finish successfully.
 
 echo ">>> RUNNING integration tests with pipeline options: $PIPELINE_OPTS"
-if [[ "$PYTEST" = true ]]; then
```

Comment:
Could you also remove the `--pytest` flag definition on line 167?

Reply:
Yes, done!